tests/src/analysis/testqgsprocessing.cpp

```diff
@@ -8998,6 +9000,135 @@ void TestQgsProcessing::parameterDxfLayers()
   QCOMPARE( dxfList.at( 0 ).layerOutputAttributeIndex(), dxfLayer.layerOutputAttributeIndex() );
 }
 
+void TestQgsProcessing::parameterPointCloudLayer()
+{
+  // setup a context
+  QgsProject p;
+  p.setCrs( QgsCoordinateReferenceSystem::fromEpsgId( 28353 ) );
+  QString testDataDir = QStringLiteral( TEST_DATA_DIR ) + '/'; //defined in CmakeLists.txt
+  QString vector1 = testDataDir + "multipoint.shp";
+  QString raster = testDataDir + "landsat.tif";
+  QString pointCloud = testDataDir + "point_clouds/las/cloud.las";
```

Comment:
Better to use the ept layers here -- they don't require the pdal dependency

Reply:
Thanks, fixed.